hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/BaseActionExecutor.java

```diff
@@ -161,4 +161,15 @@ protected final void dropIndexOnRestore() {
       }
     }
   }
+
+  protected final void disableWriteRecordPositionsWithPendingCompaction() {
```

Comment:
we also need to disable write positions when nbcc is enabled.

Reply:
fixed.